web/src/popup_banners.ts

```diff
@@ -0,0 +1,79 @@
+import $ from "jquery";
+
+import * as banners from "./banners.ts";
+import type {Banner} from "./banners.ts";
+import {$t} from "./i18n.ts";
+
+const CONNECTION_ERROR_POPUP_BANNER: Banner = {
+    intent: "danger",
+    label: $t({
+        defaultMessage: "Unable to connect to Zulip. Retrying soon...",
```

Comment:
Use the unicode `…`, not three literal `.` characters.

Reply:
Fixed, thanks!